### PR TITLE
Updated PT-BR translation

### PR DESCRIPTION
**Pull request by lucasmz-dev**
https://codeberg.org/lucasmz/translation-suggestions/issues/27

### CODE REVIEW

**Comment by lucasmz-dev** [comment]
If you prefer a single commit, you can use GitHub's squash and merge feature on the merging options. I have these three to better understand the changes I'm doing. However, that would mean my signature is lost and is replaced with GitHub. My key rotates very frequently because I change it every new OS installation so... It isn't a full verification anyway.

**Comment by lucasmz-dev** [comment]
I didn't try to build this! Sorry. Not sure how to do that. I suppose you might have actions set up to do this?

**Comment by lucasmz-dev** [comment]
Hit me up if you need updated Fastlane translations when/if? you end up changing the name here

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 1]
```diff
@@ -1,309 +1,310 @@
...
+    <string name="artists">Artistas</string>
+    <string name="albums">Álbuns</string>
     <string name="playlists">Playlists</string>
-    <string name="explore">Explore</string>
+    <string name="explore">Explorar</string>
 
     <!-- Top bar -->
     <plurals name="n_selected">
-        <item quantity="other">%d selected</item>
+        <item quantity="one">%d selecionado</item> <!-- I added this, there were no separation before, should this work automatically or should this be removed? -->
```
Comment (lucasmz-dev) [inline app/src/main/res/values-pt-rBR/strings.xml:12]: !

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 2]
```diff
@@ -1,309 +1,310 @@
...
+        <item quantity="other">%d selecionados</item>
```
Comment (lucasmz-dev) [inline app/src/main/res/values-pt-rBR/strings.xml:15]: It'd be nice to acknowledge this before this is merged.

**Comment by Malopieds** [comment]
Hi, I'm away from a keyboard for some time, thanks for the PR, but the github action seems to fail on every action anyway, I haven't properly updated it within the time...
I'll tag you when I've tried and compiled it!

**Comment by lucasmz-dev** [comment]
So cool thing, I managed to build it ! (debug and signed!) I never built any program before so that's cool. It seems alright <s>but I'll do some fine tuning on the translations.</s> (everything actually seems mostly alright, there are some strings I'm not finding though but oh well) I didn't want to have to create another PR in the future and wait for a release, so this helped a bunch.

thanks calyxos development channel for some help

**Comment by lucasmz-dev** [comment]
Expecting for #275 to be merged before this so I can deal with it better

**Comment by Malopieds** [comment]
Hi, should I merge this or prefer #344?

**Comment by lucasmz-dev** [comment]
#344 is meant to be merged first, I'll deal with this later! I'll rebase #344 real quick.

**Pull request closed without merging**